src/audio/asrc/asrc.c

```diff
@@ -0,0 +1,617 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+
+#include <sof/common.h>
+#include <sof/audio/buffer.h>
+#include <sof/audio/component.h>
+#include <sof/audio/pipeline.h>
+#include <sof/debug/panic.h>
+#include <sof/drivers/ipc.h>
+#include <sof/lib/alloc.h>
+#include <sof/lib/cache.h>
+#include <sof/list.h>
+#include <sof/math/numbers.h>
+#include <sof/platform.h>
+#include <sof/string.h>
+#include <sof/trace/trace.h>
+#include <ipc/control.h>
+#include <ipc/stream.h>
+#include <ipc/topology.h>
+#include <user/trace.h>
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <sof/audio/asrc/asrc_farrow.h>
+#include <sof/audio/format.h>
```

Comment:
Includes should go from most local to global. (and alphabetically sorted in the same level)  In this case sof -> ipc -> stdlib.

Reply:
Thanks, makes sense, my "built-in" rule was the opposite and random. I've fixed this now and applied alphabetic sort in other files.